lib/orderbook/OrderBook.ts

```diff
@@ -154,7 +218,7 @@ class OrderBook extends EventEmitter {
     const orderId = this.localIdMap.get(localId);
 
     if (orderId === undefined) {
-      throw errors.INVALID_PAIR_ID(pairId);
+      throw errors.UNSUPPORTED_PAIR(pairId);
```

Comment:
Just because the `orderId` can't be found that doesn't mean that the pair is not supported?

Reply:
Looks like this was using the wrong error message before this PR, but I'll fix it.